packages/common/src/Widget.ts

```diff
@@ -652,9 +652,12 @@ export abstract class Widget extends PropertyExt {
         });
     }
 
-    lazyRender = debounce(function (debouncedCallback?: (w: Widget) => void) {
-        this.render(debouncedCallback);
-    }, 100);
+    lazyRender(): this {
+        debounce(function (debouncedCallback?: (w: Widget) => void) {
+            this.render(debouncedCallback);
+        }, 100);
```

Comment:
This looks like if lazyRender is called multiple times within 100ms that render will also get called multiple times?

Reply:
Hmmmm, good point. How can this be avoided? Cache the callback function and compare against subsequent calls?

Reply:
...or maybe return `this` from within `debounce`? But `debounce` returns a `debounced` function. I'll give this issue another look from the perspective of rapidly calling it.

Reply:
rename to `private _lazyRender` and then have a new lazyRender call it and return this.

Reply:
@jbrundage bump